### DISCUSSION TITLE
Re-check/sync album art?

### DISCUSSION THREAD

**Discussion by Tre916** [fast4x/RiMusic#2482] [Q&A]
So, I have some tracks that for whatever reason didn't get album art pulled down, is there any way to get that to populate after the fact?

**Comment by fast4x** [comment]
If album art with correct size isn't available online, not display.

**Comment by Tre916** [comment]
I don't know if that's the issue, as it shows up in the banner and on the menu pane. Here's an example, but it's it's not the only one.
![Screenshot_20240626-085828](https://github.com/fast4x/RiMusic/assets/171737855/c3ae8f03-bd53-4f39-8556-5684eab24891)

If I had to guess, I'd guess that when it cached/downloaded the song the pull for the art timed out and it isn't updating again because.
